File: app/src/main/java/com/joker/maindexkeep/MultiProcessService.java
package com.joker.maindexkeep;

import android.app.Service;
import android.content.Intent;
import android.os.IBinder;

/**
 * manifest 中有注释
 */
public class MultiProcessService extends Service {
  public MultiProcessService() {
  }

  @Override
  public IBinder onBind(Intent intent) {
    // TODO: Return the communication channel to the service.
    throw new UnsupportedOperationException("Not yet implemented");
  }
}

File: app/src/main/java/com/joker/maindexkeep/MultiProcessService.kt
package com.joker.maindexkeep

import android.app.Service
import android.content.Intent
import android.os.IBinder

/**
 * manifest 中有注释
 */
class MultiProcessService : Service() {

  override fun onBind(intent: Intent): IBinder? {
    // TODO: Return the communication channel to the service.
    throw UnsupportedOperationException("Not yet implemented")
  }
}
